src/main.cpp

```diff
@@ -55,6 +55,42 @@ extern void qt_set_sequence_auto_mnemonic ( bool bEnable );
 
 // Implementation **************************************************************
 
+// NOTE: To prevent memory leaks any pointers assigned with new
+//       should be declared here and checked and deleted in reverse order of creation in cleanup().
+//       cleanup() should be called before any exit() !
+
+QCoreApplication* pApp       = NULL;
+CRpcServer*       pRpcServer = NULL;
+CClientRpc*       pClientRpc = NULL;
+CServerRpc*       pServerRpc = NULL;
+
+void cleanup()
+{
+    if ( pServerRpc )
+    {
+        delete pServerRpc;
+        pServerRpc = NULL;
+    }
+
+    if ( pClientRpc )
+    {
+        delete pClientRpc;
+        pClientRpc = NULL;
+    }
```

Comment:
Needs `#ifndef HEADLESS` (see Debian build failures).

Reply:
Yes! I missed that. already corrected....